nixos/tests/adguardhome.nix

```
@@ -104,32 +98,37 @@
 
   testScript = ''
     with subtest("Minimal (settings = null) config test"):
-        nullConf.wait_for_unit("adguardhome.service")
+      nullConf.wait_for_unit("adguardhome.service")
+      nullConf.wait_for_open_port(3000)
 
     with subtest("Default config test"):
-        emptyConf.wait_for_unit("adguardhome.service")
-        emptyConf.wait_for_open_port(3000)
+      emptyConf.wait_for_unit("adguardhome.service")
+      emptyConf.wait_for_open_port(3000)
+
+    with subtest("Minimal schema_version 23 (settings = null) config test"):
```

```suggestion
    with subtest("Minimal schema_version 23 config test"):
```

As `settings` is not explicitly set to `null` (and that's also not the default.